### PR TITLE
[Website] Add Jay Zhan as committer

### PR DESCRIPTION
**Pull request by alamb**
Add per https://lists.apache.org/thread/xwjg62x0bzj5bgops2yqsfzh3nqcwy73 

🎉

### CODE REVIEW

**Review by kou — APPROVED** [review]
+1

@jayzhan211 Welcome! Do you want to merge this by yourself as a new committer? You can set `affiliation` if you like.

**Comment by jayzhan211** [comment]
> https://lists.apache.org/thread/xwjg62x0bzj5bgops2yqsfzh3nqcwy73

Sure, let me do it.

**Review by jayzhan211 — APPROVED** [review]

**Comment by alamb** [comment]
🎉

File: _data/committers.yml [hunk 1]
```diff
@@ -316,6 +316,10 @@
   role: Committer
   alias: jrevels
   affiliation: Beacon Biosignals
+- name: Jay Zhan
+  role: Committer
+  alias: jayzhan
+  affiliation: None
```
[the remark this answers is not in the document]
Comment (alamb) [reply _data/committers.yml:322]: That makes total sense -- thank you for the clarification

**Pull request merged**